addons/udes_stock/models/stock_picking_type.py

```diff
@@ -513,6 +518,7 @@ def _prepare_info(self):
             - u_check_package_type: boolean
             - u_default_package_type_id: int
             - u_show_all_tasks: boolean
+            - u_enable_printing: boolean
```

Comment:
Do all the print strategies respect this if not this could cause a lot of confusion.